doc/overview.rst

```diff
@@ -725,7 +724,35 @@ retrieve the related image as follows.
        instance_type="ml.m5.xlarge",
    )
 
-Deploy a  Pre-Trained Model Directly to a SageMaker Endpoint
+Low-code Deployment
+===================
```

Comment:
How about moving `Low-code Deployment` section before `built-in Components` section?

That way, users would don't want to dive deeper into the underlying mechanics can stop reading after the `Low-code Deployment` section and try it out.

Reply:
Restructured the order based on this feedback